pkg/scheduler/framework/types.go

```diff
@@ -390,6 +390,9 @@ func (r *Resource) SetMaxResource(rl v1.ResourceList) {
 				r.MilliCPU = cpu
 			}
 		case v1.ResourceEphemeralStorage:
+			if !utilfeature.DefaultFeatureGate.Enabled(features.LocalStorageCapacityIsolation) {
+				continue
+			}
 			if ephemeralStorage := rQuantity.Value(); ephemeralStorage > r.EphemeralStorage {
 				r.EphemeralStorage = ephemeralStorage
 			}
```

Comment:
can we do it like the default case below:

```
if utilfeature.DefaultFeatureGate.Enabled(features.LocalStorageCapacityIsolation) {
  if ephemeralStorage := rQuantity.Value(); ephemeralStorage > r.EphemeralStorage {
    r.EphemeralStorage = ephemeralStorage
  }
}
```

Reply:
ok, to keep the style consistent